templates/typescript-agent/packages/web-app/.env.development

```
@@ -0,0 +1,4 @@
+MOOSE_SERVICE_URL=http://localhost:4000
+AUTH_SECRET=development-only-secret-change-me
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_SECRET key should go before the MOOSE_SERVICE_URL key

(UnorderedKey)